RussianFilter/sections/antiadblock.txt

```diff
@@ -1378,9 +1378,10 @@ music.yandex.ru,music.yandex.ua,music.yandex.by,music.yandex.kz#@%#AG_removeCook
 kinopoisk.ru#%#AG_setConstant('only_branding_adv', 'trueFunc');
 !
 drive2.ru#%#//scriptlet("set-constant", "Object.prototype._bannerOptions", "undefined")
-kinopoisk.ru,drive2.ru#%#//scriptlet('abort-on-property-read', 'Object.prototype.RtbBlockRenderAd')
-kinopoisk.ru,drive2.ru#%#//scriptlet('abort-on-property-read', 'Object.prototype.AdvManagerInit')
-kinopoisk.ru,drive2.ru#%#//scriptlet('abort-on-property-read', 'Object.prototype.AdvManagerLoaderRender')
+!
+lenta.ru,kinopoisk.ru,drive2.ru#%#//scriptlet('abort-on-property-read', 'Object.prototype.RtbBlockRenderAd')
```

Comment:
these rules do nothing on this site
All domains with this script:
`eda.ru,quto.ru,rns.online,championat.com,gazeta.ru,lenta.ru,rambler.ru`